public/whiteboard/index.html

```diff
@@ -23,12 +23,14 @@
     <script src="../js/libs/p5.min.js"></script>
     <script src="../js/utils/draggable.js"></script>
     <script src="/socket.io/socket.io.js"></script>
+    <script src="https://www.youtube.com/iframe_api"></script>
+    <script src="js/YTSampler.js" ></script>
```

Comment:
this is looking for a file called `YTSampler.js` (capital "S"), but ur file is actually called `YTsampler.js` (lower case "s"), so the js file isn't actually getting loaded up for me. i was able to change the name locally to test if it works (&& it does! solid job!) but i wonder how u got it to work on ur end w/the mis-matched file names? maybe u've got a local commit u haven't pushed yet?